fnet/fnet_model.py

```diff
@@ -20,13 +21,14 @@ def __init__(
         self.gpu_ids = [gpu_ids] if isinstance(gpu_ids, int) else gpu_ids
         
         self.criterion = criterion_fn()
-        self._init_model()
+        self._init_model(model_kwargs)
```

Comment:
this could then be **model_kwargs

Reply:
I'm okay with the _init_model() always requiring a parameter. I think `model_kwargs` should be changed to `nn_kwargs` though. It's more accurate.